docs/src/main/sphinx/develop/connectors.md

```diff
@@ -734,6 +734,10 @@ the carrier type:
 - `getSlice(int field)`
 - `getObject(int field)`
 
+Values for the `real` type are encoded into `long` using the IEEE 754
+floating-point "single format" bit layout, with NaN preservation. This
+can be accomplished using the `Float.floatToRawIntBits` static method.
```

Comment:
Maybe use the fully qualified name, so it's clear it's from the standard library:
```suggestion
can be accomplished using the `java.lang.Float.floatToRawIntBits` static method.
```